app/complaint_api.py

```diff
@@ -71,7 +73,8 @@ def post(self):
             pic_file = '%s%s' % (destination, args['sequence'] + '.jpeg')
```

Comment:
Discussed with @e-cloud, we need to refactor this API as well. Basically, we should consolidate upload_id and upload_invoice to one API, for example, upload_file. The new API accepts two parameters, first is the type of file (id, or evidence picture), and the second is pic_file. In other words, `sequence` is not necessary any longer.